exploration/04_trigger_exploration.R

```diff
@@ -0,0 +1,851 @@
+box::use(
```

Comment:
think some conceptual explanation at the top would really help a reviewer like me.... something like this i would in include:
"
This script evaluates various environmental indicators for their utility as trigger metrics in Faryab Province in AFgh. The triggers are evaluated:
- Forecasted:
    - SEAS5 – cumulative predicted rainfall for MAM predicted form Nov-March
- Observational
    - CHIRPS – Monthly rainfall: November - May
    - SWE – Monthly SWE: November – May
    - NDVI – Monthly NDVI: March- May
    - Combined Monthly CHIRPS & SWE: March- May

For each indicator we determine the worst years on record (drought-wise) across each monitoring moment/time-step. We then see how well the worst years at each of these indicator-timestep combinations line up with the worst years calculated according to ASIS Dekad 3 calculation. (include brief sentence about why ASIS may dekad 3 is used).

The heatmap & rainfall time-series produced here are included in this [slide deck](insert link)"